.travis.yml

```diff
@@ -1,19 +1,72 @@
-sudo: required
+stages:
+  - test
+  - commitlint
+  - name: release
+    if: branch = master AND type != pull_request
 
-language: python
+sudo: required
+cache: bundler
+language: ruby
 
 services:
   - docker
 
 before_install:
+  - gem install bundler
   - bundle install
 
+# Make sure the instances listed below match up with
+# the `platforms` defined in `kitchen.yml`
 env:
   matrix:
-    - INSTANCE: oxygen-cert-ubuntu-1804
-    - INSTANCE: oxygen-cert-centos-7
-    - INSTANCE: oxygen-nocerts-ubuntu-1804
-    - INSTANCE: oxygen-nocerts-centos-7
+    - INSTANCE: v2019-2-py3-debian-9
+    - INSTANCE: v2019-2-py3-ubuntu-1804
+    - INSTANCE: v2019-2-py2-centos-7
+    #- INSTANCE: v2019-2-py2-fedora-29
+
+    - INSTANCE: v2018-3-py2-debian-8
+    - INSTANCE: v2018-3-py2-ubuntu-1604
+    - INSTANCE: v2018-3-py2-bootstrap-centos-6
+    #- INSTANCE: v2018-3-py2-forced-version-fedora-28
+    - INSTANCE: v2018-3-py2-opensuse-423
+
+    - INSTANCE: v2017-7-py2-debian-8
+    - INSTANCE: v2017-7-py2-ubuntu-1604
+    - INSTANCE: v2017-7-py2-bootstrap-centos-6
```

Comment:
Can we get the new, pre-salted matrix here?

Reply:
What? Once again... sorry.

Reply:
The new pre-salted matrix from the `template-formula`.  Which was introduced with this PR: https://github.com/saltstack-formulas/template-formula/pull/92.

Reply:
@daks I'm testing this right now anyway, so the commit is available (temporarily) in my branch:

* https://github.com/myii/cert-formula/commit/5a6ecbc622457dec5cc29029b11475d6d3f6323d

This is currently running **with** Fedora, so let's see what happens with it:

* https://travis-ci.com/myii/cert-formula/builds/109793381